index.html

```diff
@@ -1,69 +1,71 @@
 ---
-# front matter tells Jekyll to process Liquid
+layout: main
 ---
 
-<!doctype html>
-<html height="100%">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+<img
+  src="{{ site.baseurl }}/assets/img/login-gov.png"
+  width="193"
+  height="26"
+  alt="login.gov logo"
+  class="margin-bottom-4"
+>
 
-    <meta name="HandheldFriendly" content="True">
-    <meta name="MobileOptimized" content="320">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<div class="grid-row grid-gap padding-bottom-1">
+  <div class="grid-col-12 desktop:grid-col-7">
+    <div class="usa-prose">
+      <h1>Verify your identity with login.gov</h1>
 
-    <title>{{ site.title }}</title>
-    <meta property="og:title" content="{{ title }}">
+      <p>If you create or renew an entity registration at SAM.gov after <strong>April 26, 2021</strong>, you will need to first verify your identity at login.gov</p>
 
-    {% if page.lead -%}
-      <meta name="description" content="{{ page.lead | strip_html }}">
-      <meta property="og:description" content="{{ page.lead | strip_html }}">
-    {% endif -%}
+      <p class="padding-top-2">To verify your identity, you’ll need:</p>
 
-    <link rel="shortcut icon" type="image/ico" href="assets/img/favicons/favicon.ico">
-    <link rel="stylesheet" href="assets/css/all-styles.css">
-    <link rel="stylesheet" href="assets/css/main.css">
-  </head>
+      <ul>
+        <li><strong>Your Social Security Number</strong></li>
+        <li><strong>A phone number on a phone plan that is in your name</strong></li>
+        <li>
+          <strong>A valid government issued ID from the following list:</strong>
+          <ul>
+            <li>Drivers license</li>
+            <li>State-issued photo ID</li>
+          </ul>
+        </li>
+      </ul>
+    </div>
 
-  <body>
-    <main class="usa-layout-docs usa-section" id="main-content">
-      <div class="grid-container">
-        <div class="usa-layout-docs__main usa-prose">
-          <img src="assets/img/sam.png" />
-          <div class="usa-display">Lorem Ipsum</div>
+    <form action="{{ site.idp_host }}{{ site.idp_path }}" method="get" class="margin-y-4">
+      <input name="client_id" value={{ site.issuer }} type="hidden" />
+      <input name="acr_values" value={{ site.acr_values }} type="hidden" />
+      <input name="redirect_uri" value={{ site.redirect_uri }} type="hidden" />
+      <input name="nonce" value={{ site.nonce }} type="hidden" />
+      <input name="state" value={{ site.state }} type="hidden" />
+      <input name="code_challenge" value={{ site.code_challenge }} type="hidden" />
```

Comment:
I see these are moved and not net-new, but WDYT of dynamically generating these in JS instead of hard coding them? Or does it not matter since we never read back the data?

Reply:
> I see these are moved and not net-new, but WDYT of dynamically generating these in JS instead of hard coding them? Or does it not matter since we never read back the data?

I checked in with @solipet to double-check. While we could generate these randomly via JavaScript, we shouldn't need to be too concerned about the specific values, since they're not actually used for anything. The user won't be directed back to this page when they finish verifying.